src/components/ha-selector/ha-selector-attribute.ts

```diff
@@ -69,11 +70,16 @@ export class HaSelectorAttribute extends LitElement {
     // Validate that that the attribute is still valid for this entity, else unselect.
     let invalid = false;
     if (this.context.filter_entity) {
-      const stateObj = this.hass.states[this.context.filter_entity];
-
-      if (!(stateObj && this.value in stateObj.attributes)) {
-        invalid = true;
-      }
+      const entityIds = ensureArray(this.context.filter_entity);
+
+      invalid = !entityIds.some((entityId) => {
+        const stateObj = this.hass.states[entityId];
+        return (
+          stateObj &&
+          this.value in stateObj.attributes &&
+          stateObj.attributes[this.value] !== undefined
+        );
+      });
```

Comment:
I think the `!` is misplaced

```suggestion
      invalid = entityIds.some((entityId) => {
        const stateObj = this.hass.states[entityId];
        return !(
          stateObj &&
          this.value in stateObj.attributes &&
          stateObj.attributes[this.value] !== undefined
        );
      });
```

Reply:
The current logic I added it : the attribute should be compatible with at least one entity. You propose that the attribute should be compatible with all the entities, right?